core/docs/spark-qualification-tool.md

```diff
@@ -49,6 +49,49 @@ Spark event logs after the application(s) have run, the second is to be integrat
 application using explicit API calls, and the third is to install a Spark listener which can output
 results on a per SQL query basis.
 
+In running the qualification tool standalone on Spark event logs, the tool can be run as a user tool command
+via a pip package for CSP environments (Google Dataproc, AWS EMR, Databricks AWS) or as a java application for
+other environments.
```

Comment:
Should we put a link to the pip package page https://pypi.org/project/spark-rapids-user-tools/ ?

Reply:
Good feedback.  Addressed in latest commit.